### PR TITLE
27 insert fitness class id in slot table

### PR DESCRIPTION
**Pull request by lamaconda**

### CODE REVIEW

File: register-service/src/main/resources/sql/20211118_08_create_slot_to_user_table.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE slot_to_user
+(
+    id      SERIAL,
+    slot_id BIGINT,
+    user_id BIGINT
+);
```
Comment (max-java) [inline register-service/src/main/resources/sql/20211118_08_create_slot_to_user_table.sql:6]: Please, add constraints on slot and user tables

File: register-service/src/main/resources/sql/20211118_06_insert_mock_data_in_fitness_class_table.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO fitness_class (fitness_class_id, fitness_class_name)
+VALUES (1, 'yoga'),
+       (2, 'power lifting'),
+       (3, 'suria namaskar');
```
Comment (max-java) [inline register-service/src/main/resources/sql/20211118_06_insert_mock_data_in_fitness_class_table.sql:4]: please add constraints on slot table

File: register-service/src/main/resources/sql/20211118_07_alter_slot_table_add_fitness_class.sql [hunk 3]
```diff
@@ -0,0 +1,4 @@
+ALTER TABLE slots ADD fitness_class_id BIGINT;
```
Comment (max-java) [inline register-service/src/main/resources/sql/20211118_07_alter_slot_table_add_fitness_class.sql:1]: please add constraints to  fitness_class_id BIGINT; not null and reference to slot table

**Review by max-java — APPROVED** [review]
minor comments, otherwise lgfm

File: register-service/src/main/resources/sql/20211118_08_create_slot_to_user_table.sql (hunk 1 above)
Comment (lamaconda) [reply register-service/src/main/resources/sql/20211118_08_create_slot_to_user_table.sql:6]: will be added later in scope of https://github.com/javaGuruBY/jis8-project/issues/44

File: register-service/src/main/resources/sql/20211118_07_alter_slot_table_add_fitness_class.sql (hunk 3 above)
Comment (lamaconda) [reply register-service/src/main/resources/sql/20211118_07_alter_slot_table_add_fitness_class.sql:1]: will be added later in scope of https://github.com/javaGuruBY/jis8-project/issues/44

File: register-service/src/main/resources/sql/20211118_06_insert_mock_data_in_fitness_class_table.sql (hunk 2 above)
Comment (lamaconda) [reply register-service/src/main/resources/sql/20211118_06_insert_mock_data_in_fitness_class_table.sql:4]: will be added later in scope of https://github.com/javaGuruBY/jis8-project/issues/44

**Pull request merged**